diff -ru libzip-1.4.0/CMakeLists.txt libzip/CMakeLists.txt
--- libzip-1.4.0/CMakeLists.txt	2017-12-29 13:37:22.000000000 +0100
+++ libzip/CMakeLists.txt	2018-03-07 01:59:08.438021912 +0100
@@ -149,10 +149,6 @@
 
 # Targets
 ADD_SUBDIRECTORY(lib)
-ADD_SUBDIRECTORY(man)
-ADD_SUBDIRECTORY(src)
-ADD_SUBDIRECTORY(regress)
-ADD_SUBDIRECTORY(examples)
 
 # pkgconfig file
 SET(prefix ${CMAKE_INSTALL_PREFIX})
diff -ru libzip-1.4.0/lib/zip.h libzip/lib/zip.h
--- libzip-1.4.0/lib/zip.h	2017-12-29 13:37:22.000000000 +0100
+++ libzip/lib/zip.h	2018-03-06 23:28:15.816486500 +0100
@@ -46,9 +46,9 @@
 
 #ifndef ZIP_EXTERN
 # ifndef ZIP_STATIC
-#  ifdef _WIN32
+#  ifdef _MSC_VER
 #   define ZIP_EXTERN __declspec(dllimport)
-#  elif defined(__GNUC__) && __GNUC__ >= 4
+#  elif (defined(__GNUC__) && __GNUC__ >= 4) && !defined(_WIN32)
 #   define ZIP_EXTERN __attribute__ ((visibility ("default")))
 #  else
 #   define ZIP_EXTERN
diff -ru libzip-1.4.0/libzip.pc.in libzip/libzip.pc.in
--- libzip-1.4.0/libzip.pc.in	2017-12-29 13:37:22.000000000 +0100
+++ libzip/libzip.pc.in	2018-03-07 01:59:38.626003683 +0100
@@ -8,6 +8,7 @@
 Name: libzip
 Description: library for handling zip archives
 Version: @VERSION@
-Libs: @PKG_CONFIG_RPATH@ -L${libdir} -lzip @LIBS@
+Libs: @PKG_CONFIG_RPATH@ -L${libdir} -lzip
+Libs.private: -lbz2
 Cflags: -I${includedir}
 
