From 816a16f9c52417e82895bfbc8eac40124c7328d7 Mon Sep 17 00:00:00 2001
From: Martin Herkt <lachs0r@srsfckn.biz>
Date: Wed, 14 Feb 2018 10:09:12 +0100
Subject: [PATCH 2/3] always include install rules

---
 CMakeLists.txt        | 4 ----
 loader/CMakeLists.txt | 2 +-
 2 files changed, 1 insertion(+), 5 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 5cf85d48..0ad37985 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -362,7 +362,6 @@ run_external_revision_generate(${EXTERNAL_SOURCE_ROOT}/glslang/External/spirv-to
 
 
 
-if(NOT WIN32)
     include(GNUInstallDirs)
 
     add_definitions(-DFALLBACK_CONFIG_DIRS="${FALLBACK_CONFIG_DIRS}")
@@ -373,9 +372,7 @@ if(NOT WIN32)
     if(NOT (CMAKE_INSTALL_FULL_SYSCONFDIR STREQUAL "/etc"))
         add_definitions(-DEXTRASYSCONFDIR="/etc")
     endif()
-endif()
 
-if(UNIX)
     if(INSTALL_LVL_FILES)
         install(DIRECTORY "${PROJECT_SOURCE_DIR}/include/vulkan" DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
         install(FILES "${CMAKE_CURRENT_BINARY_DIR}/vk_layer_dispatch_table.h" DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}/vulkan")
@@ -389,7 +386,6 @@ configure_file(
 
 add_custom_target(uninstall
     COMMAND ${CMAKE_COMMAND} -P ${CMAKE_CURRENT_BINARY_DIR}/cmake_uninstall.cmake)
-endif()
 
 # Layer Utils Library
 # For Windows, we use a static lib because the Windows loader has a fairly restrictive loader search
diff --git a/loader/CMakeLists.txt b/loader/CMakeLists.txt
index fb626b63..e2780431 100644
--- a/loader/CMakeLists.txt
+++ b/loader/CMakeLists.txt
@@ -225,7 +225,7 @@ else()
     target_link_libraries(${API_LOWERCASE} -ldl -lpthread -lm)
 
     if(INSTALL_LVL_FILES)
-        install(TARGETS ${API_LOWERCASE} LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR})
+        install(TARGETS ${API_LOWERCASE} LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR} ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR})
     endif()
 
     # Generate pkg-config file.
-- 
2.16.1

